src/launcher/features/sources/AddLegacySourceWarning.tsx

```diff
@@ -32,13 +32,13 @@ export default () => {
             }}
         >
             <p>
-                Adding sources on <code>developer.nordisemi.com</code> is no
+                Adding sources from <code>developer.nordisemi.com</code> is no
                 longer supported.
             </p>
             <p>
-                Instead of adding the source with the URL{' '}
-                <code>{sourceToAdd}</code> a source with the URL{' '}
-                <code>{correctedUrl}</code> can be added.
+                For the URL you&apos;ve provided{' '}
```

Comment:
```suggestion
                For the URL you&apos;ve provided
```

The space at the end is wrong. But I like anyhow that you removed `sourceToAdd` from the message and will remove the space as well as the commented out section in an own PR, so there is probably no need to fix this by itself.